src/main/DAL/MetaQuestion/Appendix.js

```diff
@@ -7,15 +7,16 @@ function defineAppendixModel(sequelize) {
     return sequelize.modelManager.models.some(model => model.name === 'Appendix') ?
         sequelize.models.Appendix :
         sequelize.define('Appendix', {
-                title: {
+                tag: {
```

Comment:
Is tag should represent id? and if so why not to call it id?

Reply:
sole reason is that's how Mayer originally sectioned the appendix: tag, title, content
where tag is unique